client/components/place/place-list/place-list.template.html

```diff
@@ -38,12 +38,12 @@
               <span class="caret"></span>
             </button>
             <ul class="dropdown-menu col-xs-12 dropdownFilter">
-              <li ng-repeat="type in $ctrl.tracksType" ng-click="$ctrl.showSpecificTracks(type.type)">
-                <a href=""  class="active" id="{{type.type}}">
+              <li ng-repeat="type in $ctrl.tracksType" ng-click="$ctrl.showSpecificTracks(type.name)">
```

Comment:
https://docs.angularjs.org/api/ng/directive/ngRepeat
`ng-repeat="(type, typeObj) in $ctrl.tracksType"`
`ng-click="$ctrl.showSpecificTracks(type)"`